src/core/config/ip6.h

```diff
@@ -177,4 +177,14 @@
 #define OPENTHREAD_CONFIG_UNSECURE_TRAFFIC_MANAGED_BY_STACK_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_TCP_ENABLE
+ *
+ * Define as 1 to enable TCP.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_TCP_ENABLE
+#define OPENTHREAD_CONFIG_TCP_ENABLE 1
```

Comment:
I think this is good here now, in future I expect we probably will have more configs for TCP module, we can then add them all in a new header `config/tcp.h`.

Reply:
Noted.